File: lua/refactoring/tests/refactor/106/java/prompt_func_return/extract.expected.java
class Extract {

public static int foo() {
        System.out.println("this is a test");
        int test = 1;
        return test;
}


    public static int simpleFunction(int a) {
        var test = foo();

        System.out.println(test);

        return test;
    }
}

File: lua/refactoring/tests/refactor/106/csharp/prompt_func_return/extract.expected.cs
namespace Test
{
    class Extract
    {
        public static int foo()
        {
            Console.WriteLine("this is a test");
            int test = 1;
            return test;
        }


        public static int simpleFunction(int a)
        {
            var test = foo();

            Console.WriteLine(test);

            return test;
        }
    }
}

